docs/Dev_env_setup.rst

```diff
@@ -0,0 +1,116 @@
+Development environment setup
+===============================
+
+Here are some guidelines on how to set up your development environment for the 
+lago project.
+
+
+Requirements
+-------------
+
+You'll need some extra packages to get started with the code for lago, assuming
+you are runnig fedora::
+
+  > sudo dnf install git mock libvirt-daemon qemu-kvm autotools
+
+And you'll need also a few python libs, you can install them from the repos or
+use venv or similar, for the sake of this example we will use the repos ones::
+
+  > sudo dnf install python-flake8 python-nose python-dulwich yapf
+
+Yapf is not available on older fedoras or CentOS, you can get it from the
+`official yapf repo`_ or try on `copr`_.
+
+Now you are ready to get the code::
+
+  > git clone git://gerrit.ovirt.org/lago
+
+From now on all the commands will be based from the root of the cloned repo::
+
+  > cd lago
+
+
+Style formatting
+------------------
+
+We will not accept any patch that does not pass pep8 and that it's not formatted
```

Comment:
Change to a positive tone: We accept patches that pass PEP8 and YAPF formatting only.